test/test_events_le.py

```diff
@@ -66,10 +73,10 @@ def setUp(self):
             'type': 173,
             'data': [130, 181, 177, 244, 167, 144, 216, 185, 228, 220, 254, 130, 239, 213, 142, 240, 233, 248, 161, 238, 160, 205, 212, 205, 166, 156, 241, 169, 198, 147, 144],
             }
-        self.evt_bin_0 = '\xad\x82\xb5\xb1' '\xf4\xa7\x90\xd8' \
-            '\xb9\xe4\xdc\xfe' '\x82\xef\xd5\x8e' \
-            '\xf0\xe9\xf8\xa1' '\xee\xa0\xcd\xd4' \
-            '\xcd\xa6\x9c\xf1' '\xa9\xc6\x93\x90'
+        self.evt_bin_0 = b'\xad\x82\xb5\xb1' b'\xf4\xa7\x90\xd8' \
```

Comment:
The diff from here onwards is a find and replace to add the `b` prefix for bytes literals.